### DISCUSSION TITLE
Expected traffic share of the privacy sandbox OT

### DISCUSSION THREAD

**Discussion by alois-bissuel** [privacysandbox/privacy-sandbox-dev-support#31] [Replacing cross-site tracking] [answered]
Hi,

I have just read your explainer about the [unified OT of the privacy sandbox](https://developer.chrome.com/blog/privacy-sandbox-unified-origin-trial/#eligible-users), and there are two points which caught my attention:
1. only users on **beta** versions of chrome would be eligible
2. users would need to activate the privacy sandbox flags (namely chrome://flags/#privacy-sandbox-ads-apis)

From what I can read, this does not look very different from the [requirements for local testing ](https://developer.chrome.com/blog/privacy-sandbox-unified-origin-trial/#local-testing).

Is this expected? 

Follow-up question: if the OT is activated on stable traffic, what is the expected traffic share on which it will be activated? Less than 1%, less than 5%? Will this evolve over time?

Thanks a lot!

**Comment by samdutton** [comment]
Apologies for the confusion here — we'll update the post to clarify.

If you include a valid trial token in a page, all Chrome Beta 101 users on that page are eligible for trial features to be enabled, without them needing to adjust flags (though bear in mind [What users are eligible for the origin trial?](https://developer.chrome.com/blog/privacy-sandbox-unified-origin-trial/#eligible-users))

Enabling `chrome://flags/#privacy-sandbox-ads-apis` makes the origin trial features available to any individual user. (This is intended for developers trying out the APIs locally, not for testing at scale.) This flag does not need to be enabled for a user to be eligible for origin trial features on pages that provide a valid token.

I'll get back to you on the follow-up question.

**Reply by ankwok** [reply]
I'd like to add another follow-up question: Are all three OT features activated as a group?  For example, could I ever encounter a browser where FLEDGE is activated but not Topics?

**Reply by samdutton** [reply]
@ankwok — that's correct: if a browser [is part of the active experiment group in the origin trial](https://developer.chrome.com/blog/privacy-sandbox-unified-origin-trial/#eligible-users), all three APIs will be available to the browser on any page that provides a valid trial token (and vice versa).

**Comment by alois-bissuel** [comment, marked as answer]
Hello @samdutton,

Would you have news regarding my other question, namely the projected share of traffic on standard Chrome version? For now we don't see a lot of traffic (apparently there are not so many users of Chrome Beta out there).

Thanks a lot!

**Reply by samdutton** [reply]
Hi @alois-bissuel ! 

Just realised I never got back to you...

[This post](https://groups.google.com/a/chromium.org/g/attribution-reporting-api-dev/c/Zqeq6UPP5wk/m/LJ-euyRtAgAJ) has a pretty good summary of the current status and plans.

**Reply by alois-bissuel** [reply]
Thanks for the answer, and closing the question.
